core/src/com/softwarearchitecture/ECS/Components/EnemyComponent.java

```diff
@@ -0,0 +1,9 @@
+package com.softwarearchitecture.ecs.components;
+
+public class EnemyComponent {
+    public int damage;
+
+    public EnemyComponent(int damage) {
+        this.damage = damage;
+    }
```

Comment:
I think it could be a good idea to add a check if the damage is set to a negative value, in this way you cannot heal from an attack. (Unless we want to add such an ability?)